rpc/types_trace.go

```diff
@@ -39,28 +39,31 @@ var _ TxnTrace = L1HandlerTxnTrace{}
 type InvokeTxnTrace struct {
 	ValidateInvocation FnInvocation `json:"validate_invocation"`
 	//the trace of the __execute__ call or constructor call, depending on the transaction type (none for declare transactions)
-	ExecuteInvocation     ExecInvocation  `json:"execute_invocation"`
-	FeeTransferInvocation FnInvocation    `json:"fee_transfer_invocation"`
-	StateDiff             StateDiff       `json:"state_diff"`
-	Type                  TransactionType `json:"type"`
+	ExecuteInvocation     ExecInvocation     `json:"execute_invocation"`
+	FeeTransferInvocation FnInvocation       `json:"fee_transfer_invocation"`
+	StateDiff             StateDiff          `json:"state_diff"`
+	Type                  TransactionType    `json:"type"`
+	ExecutionResources    ExecutionResources `json:"execution_resources"`
```

Comment:
There is no usage or test that I can see for this addition.

Reply:
Will try and get test data from Juno

Reply:
Later commit adds test

Reply:
TestInvokReceipt

Reply:
updated TestTransactionTrace to use a trace from RPCv07-rc2